src/lib/components/dateSelector/index.test.tsx

```diff
@@ -14,40 +14,138 @@ const setup = (date: string, onChange: (date: string) => void = () => {}) => {
 };
 
 describe('DateSelector component', () => {
-  it('should return the selected date when clicking on the calendar', async () => {
+  it('should show the value inside the inputs', () => {
+    const date = '2024-01-01';
+    const { getByTestId } = setup(date);
+
+    expect(getByTestId('date-selector-day')).toHaveValue('1');
+    expect(getByTestId('date-selector-month')).toHaveValue('1');
+    expect(getByTestId('date-selector-year')).toHaveValue('2024');
+  });
+
+  it('should call onChange with the value changed', async () => {
+    const callback = jest.fn();
+    const { getByTestId, user } = setup(undefined, callback);
+
+    await user.type(getByTestId('date-selector-day'), '5');
+    await user.type(getByTestId('date-selector-month'), '7');
+    await user.type(getByTestId('date-selector-year'), '2023');
```

Comment:
To make sure we are not impacting accessibility, IMHO it'd be better if we stick to the actual input labels, and not rely on `data-testid`.

Is there a reason to use `data-testid` to target these inputs?